src/action_cc.cc

```diff
@@ -552,6 +552,73 @@ NM_ACTION_(nickel_wifi) {
     return nm_action_result_silent();
 }
 
+NM_ACTION_(nickel_orientation) {
+    Qt::ScreenOrientation o;
+
+    if      (!strcmp(arg, "portrait"))           o = Qt::PortraitOrientation;
+    else if (!strcmp(arg, "landscape"))          o = Qt::LandscapeOrientation;
+    else if (!strcmp(arg, "inverted_portrait"))  o = Qt::InvertedPortraitOrientation;
+    else if (!strcmp(arg, "inverted_landscape")) o = Qt::InvertedLandscapeOrientation;
+    // TODO: maybe "invert", "rotate_cw", "rotate_ccw" options.
```

Comment:
...

Reply:
The current implementation would be my preferred wording, too, FWIW ;).

Reply:
Current implementation?

Reply:
i.e., I don't think adding aliases will make it clearer, possible quite the contrary ;).

Reply:
I was mainly thinking of `invert` to flip the current orientation. That would be useful for two things:
- Not cluttering up the reading menu or having duplicates between that and the actual rotation menu.
- Having an easy way to make use of the fact that the actual rotation menu's locked portrait/landscape allows for both normal and inverted versions.

But yes, I do agree that `rotate_cw` and `rotate_ccw` are probably unnecessary.

Reply:
Oh, right, yeah, an automagic `invert` does sound cool then, yeah ;).

Reply:
Which might entail an automagic `swap` or something if the quirks of Portrait <-> Landscape swaps aren't too gnarly in practice?

Reply:
The complication with portrait/landscape swaps is that I may need to change the locked orientation too (which will already be done if I implement it for the comment below). I just need to figure out the simplest version-agnostic way to get ahold of the RotatePopupController.

Reply:
Done.